react/src/lib/components/DeckGLMap/SubsurfaceViewerDashWrapper.tsx

```diff
@@ -0,0 +1,278 @@
+    views?: ViewsType;
+    coords?: {
+        visible?: boolean | null;
+        multiPicking?: boolean | null;
+        pickDepth?: number | null;
+    };
+    scale?: {
+        visible?: boolean | null;
+        incrementValue?: number | null;
+        widthPerUnit?: number | null;
+        cssStyle?: Record<string, unknown> | null;
+    };
+    coordinateUnit?: string;
+    colorTables?: colorTablesArray;
+    editedData?: Record<string, unknown>;
+    setProps?: (data: Record<string, unknown>) => void;
+
+    /**
+     * Validate JSON datafile against schema
+     */
+    checkDatafileSchema?: boolean;
+
+    /**
+     * For get mouse events
+     */
+    onMouseEvent?: (event: MapMouseEvent) => void;
+
+    getCameraPosition?: (input: ViewStateType) => void;
+
+    /**
+     * If changed will reset camera to default position.
+     */
+    triggerHome?: number;
+    triggerResetMultipleWells?: number;
+    /**
+     * Range selection of the current well
+     */
+    selection?: {
+        well: string | undefined;
+        selection: [number | undefined, number | undefined] | undefined;
+    };
+
+    /**
+     * Override default tooltip with a callback.
+     */
+    getTooltip?: TooltipCallback;
+    cameraPosition?: ViewStateType | undefined;
+
+    children?: React.ReactNode;
+
+    /**
+     * A mapping associating annotation components to view ids.
+     * Example: {"view_1": <ColorLegend/>}
+     */
+    annotation?: Record<string, unknown>;
+}
+
+function mapAnnotation(annotation: Record<string, unknown>) {
+    return Object.entries(annotation).map(([viewId, annotation]) => (
+        <View key={viewId} id={viewId}>
+            {annotation}
+        </View>
+    ));
+}
+
+const SubsurfaceViewerDashWrapper: React.FC<
+    SubsurfaceViewerDashWrapperProps
+> = ({
+    id,
+    resources,
+    layers,
+    bounds,
+    views,
+    coords,
+    scale,
+    coordinateUnit,
+    colorTables,
+    editedData,
+    setProps,
+    checkDatafileSchema,
+    onMouseEvent,
+    selection,
+    getTooltip,
+    cameraPosition,
+    getCameraPosition,
+    triggerHome,
+    triggerResetMultipleWells,
+    children,
+    annotation = {},
+}: SubsurfaceViewerDashWrapperProps) => {
+    const mapArgs: DeckGLMapProps = {
+        id: id,
+        resources: resources,
+        layers: layers,
+        bounds: bounds,
+        views: views,
+        coords: coords,
+        scale: scale,
+        coordinateUnit: coordinateUnit,
+        colorTables: colorTables,
+        editedData: editedData,
+        setProps: setProps,
+        checkDatafileSchema: checkDatafileSchema,
+        onMouseEvent: onMouseEvent,
+        selection: selection,
+        getTooltip: getTooltip,
+        cameraPosition: cameraPosition,
+        getCameraPosition: getCameraPosition,
+        triggerHome: triggerHome,
+        triggerResetMultipleWells: triggerResetMultipleWells,
+        children: children,
+    };
+
+    return (
+        <>
```

Comment:
Think we do not need the <></> tag here as we do not need to wrap any JSX element.